tests/semanticdb/metac.expect

```diff
@@ -84,7 +86,9 @@ advanced/Test.s3x. => val method s3x Int
 advanced/Wildcards# => class Wildcards extends Object { self: Wildcards => +3 decls }
 advanced/Wildcards#`<init>`(). => primary ctor <init> (): Wildcards
 advanced/Wildcards#e1(). => method e1 => List[_] forSome { type _  }
+advanced/Wildcards#e1().[_] => type _ 
```

Comment:
Sorry for all the complication, but I think you should only register `WildcardTypeSymbol` in the symbol section when it is the parameter of `TypeBounds`, and we are not inside the arguments of an `AppliedType` - as soon as you are in an `AppliedType` then the wildcard is local to the `ExistentialType` that binds it. For example a lot of no arg methods, or even vals, are getting these type parameters